### PR TITLE
update push workflow and hide data panel collapse button

### PR DESCRIPTION
**Pull request by Nirmay15**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "lint": "eslint --ext .tsx,.ts packages/**",
     "clean": "rm -rf packages/**/lib packages/**/dist packages/**/build",
-    "build-all": "nx run-many --target=build"
+    "build-all": "nx run-many --target=build",
+    "build-dist-all":"nx run-many --target=dist"
```
Comment (ashubham) [inline package.json:9]: lets call this `dist-all`

File: packages/sheets/package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "copy-index": "gsuite-shell"
   },
   "dependencies": {
-    "@thoughtspot/visual-embed-sdk": "^1.19.0-alpha.6",
+    "@thoughtspot/visual-embed-sdk": "^1.19.0-alpha.7",
```
Comment (ashubham) [inline packages/sheets/package.json:29]: I see this version is not pushed to github for visual-embed-sdk. Lets make sure we publish new version of the visual-embed-sdk only after we push the code to github.

File: .github/workflows/push_to_plugins.yml [hunk 3]
```diff
@@ -35,7 +35,7 @@ jobs:
 
       # Runs the build for all sub projects
       - name: Run build
-        run: npm run build-all
+        run: npm run build-all && HOST=${{ vars.HOST }} npm run build-dist-all
```
Comment (ashubham) [inline .github/workflows/push_to_plugins.yml:38]: Lets split this into two steps instead of using &&

**Review by ashubham — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Nirmay15) [reply package.json]: Done

File: packages/sheets/package.json (hunk 2 above)
Comment (Nirmay15) [reply packages/sheets/package.json]: Ack

File: .github/workflows/push_to_plugins.yml (hunk 3 above)
Comment (Nirmay15) [reply .github/workflows/push_to_plugins.yml]: Done

**Review by ashubham — APPROVED** [review]

**Pull request merged**